bots/cli/src/main/java/org/openjdk/skara/bots/cli/BotLogstashHandler.java

```diff
@@ -33,12 +33,10 @@
 import java.util.*;
 import java.util.logging.*;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
-public class BotLogstashHandler extends BotTaskAggregationHandler {
+public class BotLogstashHandler extends StreamHandler {
     private final RestRequest endpoint;
```

Comment:
One thing that I realized when thinking about this, is that this change will result in a lot more POSTs. However, RestRequest implements various rate limits to adhere to GitHub policies like https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting and https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-abuse-rate-limits . So it would perhaps be better to just use HttpClient directly in this context.